docker-compose.yml

```diff
@@ -1,5 +1,52 @@
 version: "3"
 services:
+  frontend:
+    image: quay.io/hmpps/hmpps-approved-premises-ui:latest
+    container_name: approved-premises-ui-dev
+    environment:
+      - REDIS_HOST=approved-premises-redis-dev
+      - REDIS_PORT=6379
+      - REDIS_TLS_ENABLED=false
+      - REDIS_AUTH_TOKEN=
+      - HMPPS_AUTH_EXTERNAL_URL=http://localhost:9091/auth
+      - HMPPS_AUTH_URL=http://hmpps-auth:8080/auth
+      - API_CLIENT_ID=approved-premises-ui
+      - API_CLIENT_SECRET=clientsecret
+      - SYSTEM_CLIENT_ID=approved-premises-ui
+      - SYSTEM_CLIENT_SECRET=clientsecret
+      - SESSION_SECRET=app-insecure-default-session
+      - NODE_ENV=development
+      - TOKEN_VERIFICATION_API_URL=http://hmpps-auth:8080/verification
+      - APPROVED_PREMISES_API_URL=http://host.docker.internal:8080
+      - INGRESS_URL=http://localhost:3000
+    ports:
+      - 3000:3000
+    entrypoint: "node dist/server.js | bunyan"
+
+    frontend-remote-debug:
+      image: quay.io/hmpps/hmpps-approved-premises-ui:latest
+      container_name: approved-premises-ui-dev
+      environment:
+        - REDIS_HOST=approved-premises-redis-dev
+        - REDIS_PORT=6379
+        - REDIS_TLS_ENABLED=false
+        - REDIS_AUTH_TOKEN=
+        - HMPPS_AUTH_EXTERNAL_URL=http://localhost:9091/auth
+        - HMPPS_AUTH_URL=http://hmpps-auth:8080/auth
+        - API_CLIENT_ID=approved-premises-ui
+        - API_CLIENT_SECRET=clientsecret
+        - SYSTEM_CLIENT_ID=approved-premises-ui
+        - SYSTEM_CLIENT_SECRET=clientsecret
+        - SESSION_SECRET=app-insecure-default-session
+        - NODE_ENV=development
+        - TOKEN_VERIFICATION_API_URL=http://hmpps-auth:8080/verification
+        - APPROVED_PREMISES_API_URL=http://host.docker.internal:8080
+        - INGRESS_URL=http://localhost:3000
+      ports:
+        - 3000:3000
+        - 9229:9229
+      entrypoint: "node --inspect-brk=0.0.0.0 dist/server.js | bunyan"
```

Comment:
I think this either needs to be commented out or shifted right if we want to run this container also?

Reply:
right you are!  Not sure what happened there, have sorted